tests/phpunit/tests/l10n/wpLocaleSwitcher.php

```diff
@@ -583,6 +591,63 @@ public function test_switch_to_locale_should_work() {
 		$this->assertSame( 'Este es un plugin dummy', $actual_es_es );
 	}
 
+
+	/**
+	 * @covers ::switch_to_locale
+	 * @covers ::switch_to_user_locale
+	 * @covers WP_Locale_Switcher::get_current_locale
+	 * @covers WP_Locale_Switcher::get_current_user_id
+	 */
+	public function test_returns_current_locale_and_user_after_switching() {
+		global $wp_locale_switcher;
+
+		$user_2 = self::factory()->user->create(
+			array(
+				'role'   => 'administrator',
+				'locale' => 'es_ES',
+			)
+		);
+
+		$locale_1  = $wp_locale_switcher->get_current_locale();
+		$user_id_1 = $wp_locale_switcher->get_current_user_id();
+
+		switch_to_user_locale( self::$user_id );
+
+		$locale_2  = $wp_locale_switcher->get_current_locale();
+		$user_id_2 = $wp_locale_switcher->get_current_user_id();
+
+		switch_to_locale( 'en_GB' );
+
+		$locale_3  = $wp_locale_switcher->get_current_locale();
+		$user_id_3 = $wp_locale_switcher->get_current_user_id();
+
+		switch_to_user_locale( $user_2 );
+
+		$locale_4  = $wp_locale_switcher->get_current_locale();
+		$user_id_4 = $wp_locale_switcher->get_current_user_id();
+
+		restore_current_locale();
+
+		$locale_5  = $wp_locale_switcher->get_current_locale();
+		$user_id_5 = $wp_locale_switcher->get_current_user_id();
+
+		$this->assertFalse( $locale_1 );
```

Comment:
Can you add another parameter to the end of each of the assertions to describe what went wrong should one of them fail?

That way, contributors can see _which_ of the `assertFalse()` or `assertSame()` failed, with some added context.

e.g.

`$this->assertFalse( $locale_1, 'Locale should be false before switching.' );`

`$this->assertSame( 'de_DE', $locale_2, 'The locale was not changed to de_DE.' );`

and so on.

Note: This is required as documented in the [Writing PHPUnit Tests - Using Assertions entry](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions) handbook entry:
> All PHPUnit assertions, as well as all WordPress custom assertions, allow for a `$message` parameter to be passed. This message will be displayed when the assertion fails and can help immensely when debugging a test. This parameter should always be used if more than one assertion is used in a test method.